pkgdb/src/registry/floxpkgs.cc

```diff
@@ -517,8 +517,9 @@ FloxFlakeScheme::inputFromAttrs( const nix::fetchers::Attrs & _attrs ) const
     = toGitHubAttrs( attrs );
   auto                                ghAttrs  = ghAttrsAndOurAttrs.first;
   auto                                ourAttrs = ghAttrsAndOurAttrs.second;
+  GitHubInputScheme                   githubScheme;
   std::optional<nix::fetchers::Input> fromGithub
-    = GitHubInputScheme::inputFromAttrs( ghAttrs );
+    = githubScheme.inputFromAttrs( ghAttrs );
```

Comment:
This here was the reason the `schema` part was always `null`.

```c++
GitHubInputScheme::inputFromAttrs( ghAttrs );
```

resolves to "flox-nixpkgs" in this static context.
which means you always run into

```c++
if ( ! fromGithub.has_value() ) { return std::nullopt; }
```

below, which in turn meant the flakeref was initialized without a schema

You have to create an instance of the schema first and then call `inputFromAttrs` as an instance method for `this` to be resolved correctly.